004-directives/src/app/components/card/card.directive.js

```diff
@@ -8,7 +8,27 @@
   /** @ngInject */
   function cardDirective() {
     return {
-      templateUrl: 'app/components/card/card.html'
+      restrict: 'E',
+      templateUrl: 'app/components/card/card.html',
+      controller: 'CardController',
+      controllerAs: 'cardCtrl',
+      scope: {
+        // String representing the icon to use in the card.
```

Comment:
this is ok to explain the directive, but next time try to use docs http://usejsdoc.org/